lib/routes/routeBackbeat.js

```diff
@@ -151,10 +176,59 @@ function putMetadata(request, response, bucketInfo, objMd, log, callback) {
     });
 }
 
+function putDataMultipleBackends(request, response, bucketInfo, objMd, log,
+    callback) {
+    const storageLocation = request.headers['x-scal-storage-class'];
+    const canonicalID = request.headers['x-scal-canonical-id'];
+    if (canonicalID === undefined) {
```

Comment:
what if canonicalID (and the headers content-md5 & scal-version-id below) is an empty string?
is some validation for format performed later on?

Reply:
I assume the values stored in object metadata are valid, so we just need to check that the header is sent.